Additional Cubeviz docs sections

rosteen opened the pull request:
Fills in some more sections of the Cubeviz docs.

javerbukh commented on docs/cubeviz/displaycubes.rst:
```diff
@@ -27,11 +29,18 @@ also be collapsed into a spectrum using the same statistic and displayed in
 
-Coming soon
+The data to be displayed in each viewer can be selected and de-selected by 
+clicking on the "hammer and screwdriver" icon at the top left of each viewer. 
```
Patrick suggested using :guilabel:\`Hammer and screwdriver\` to describe UI elements.

rosteen replied:
Good call, will push an update soon

javerbukh commented on docs/cubeviz/displaycubes.rst:
```diff
@@ -27,11 +29,18 @@ also be collapsed into a spectrum using the same statistic and displayed in
-Coming soon
+The data to be displayed in each viewer can be selected and de-selected by 
+clicking on the "hammer and screwdriver" icon at the top left of each viewer. 
+Then, click the "gear" icon to access the "Data" tab. Here, you can click the 
```
Same as above.

javerbukh commented on docs/cubeviz/import_data.rst:
```diff
@@ -1,5 +1,43 @@
+------------------------------
+
+The first way that users can load their data into the Cubeviz application is 
+by clicking the "Import" button at the top left of the application's user interface. 
```
Same as above.

javerbukh commented on docs/cubeviz/import_data.rst:
```diff
@@ -1,5 +1,43 @@
+
+.. image:: img/cubeviz_import_data.png
+
+After clicking "Import", the data file will be parsed and loaded into the 
```
Here as well.

javerbukh commented on docs/cubeviz/import_data.rst:
```diff
@@ -1,5 +1,43 @@
+
+After clicking "Import", the data file will be parsed and loaded into the 
+application. A notification will appear to let users know if the data import 
+was successful. Afterward, the new data set can be found in the "Data" tab 
```
And here.

javerbukh reviewed (approved):
Left a couple comments but otherwise looks good!

rosteen commented:
I just rebased to the traitlets fix on master to (hopefully) get the tests passing, and squashed most of the commits for a cleaner history. Should be good to merge once the tests pass.

rosteen commented:
Everything passing but the know Windows CI problem

javerbukh commented:
Merging.
